test/contains.js

```diff
@@ -28,6 +28,36 @@ describe('contains', function() {
     assert.strictEqual(R.contains(new Just([42]), [new Just([42])]), true);
   });
 
+  it('dispatches to `indexOf` method', function() {
```

Comment:
Are you suggesting this be an undocumented feature which we're free to remove at any time?

Derivability is mentioned in the Fantasy Land specification in several places. [Applicative][1], for example, does not require `map` as it can be derived from `of` and `ap`:

> A value which satisfies the specification of an Applicative does not need to implement:
>
>   - Functor's `map`; derivable as `function(f) { return this.of(f).ap(this); })}`

We could *also* dispatch to `contains`, but it seems completely reasonable to me to state that a type which provides `indexOf` but not `contains` is compatible with `R.contains`.


[1]: https://github.com/fantasyland/fantasy-land#applicative